examples/apps/djapp/legacy/4_create_initial_mesh_components/mesh.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Does this work with legacy crds?

Reply:
more cruft, good catch! sorted now, i checked master out and diff'd against all originals. these were just placeholders for myself to fix - before i just wrote all new yaml for v1beta2. sorted now, legacy is back to square, with the exception of adding 'namespace' in some. thanks!